.storybook/custom/components/Header.js

```diff
@@ -18,7 +18,7 @@ const Header = () => {
                 </a>
                 <a
                 className='fddocs-header__anchor'
-                href={`https://github.com/SAP/fundamental-styles/tree/v${packageJson.version}`}
```

Comment:
The whole point of this is to take you to the version that you're seeing docs for though - could we create a separate link for previous versions?

Reply:
Or better yet, add a dropdown that has previous version links in it?

Reply:
I don't see why linking someone to the tag for this specific version would be very useful, figured this would be more helpful.

Not sure what dropdown we could use here that would fit stylisticly so I just went with the link

Reply:
The logic behind it is: say you're looking at docs for version `4.0.0`, when you click the link, you're going to want to see the code from `4.0.0`. If master is on `4.0.0-rc.20`, you don't want that because the code on master may no longer match the documentation you were looking at. 

If we want to support multiple versions, we should just set that up in general. The whole point of being 0th version is to not have to support old versions and be able to move quickly. Checkout out the Concur docs for how we supported multiple versions - https://hig.concur.com/pages/develop/web/components/nui-widgets/v13/card.html

Reply:
If we do anything, I would prefer to see a docs site version switcher.  Ultimately, it would be nice if this was automated as part of the build/release process, but is there a way we could make a dropdown list of the older versions (even if it's just the hard-coded links from the wiki page)?  Either build the current version tag directly into it or leave this version tag and put an "Other Releases" dropdown next to it.  Although a wiki page is nice and quick, it's also kind of kludgy.

Reply:
I like Greg's Idea of having `Other Releases` dropdown - fd-select, or native select, similar to NGX.